src/job/models/newJobHandler.ts

```diff
@@ -55,7 +55,6 @@ export class NewJobHandler extends JobHandler implements IJobHandler {
       logger.info({ msg: 'building tasks' });
       const mergeTasks = this.taskBuilder.buildTasks(taskBuildParams);
 
-      logger.info({ msg: 'pushing tasks' });
```

Comment:
Please delete also for other job handlers

Reply:
done